### DISCUSSION TITLE
How do I enable uninstall feature in latest preview of winget v-0.2.10191

### DISCUSSION THREAD

**Discussion by neeraj9** [microsoft/winget-cli#735] [Q&A] [answered]
WinGet is great for community (like me) who are used to command line pretty much for everything. Thanks for sharing this.
I noticed that the latest preview [1] has an uninstall feature, but its not clear what should I be doing to enable it. I cannot see it in my regular winget command execution. Additionally, when I try to use the ```winget settings``` command there is no clue about the settings required for uninstall either.

```console
Windows Package Manager v0.2.10191 Preview
Copyright (c) Microsoft Corporation. All rights reserved.

The winget command line utility enables installing applications and other packages from the command line.

usage: winget [<command>] [<options>]

The following commands are available:
  install   Installs the given package
  show      Shows info about an package
  source    Manage sources of packages
  search    Find and show basic info of packages
  hash      Helper to hash installer files
  validate  Validates a manifest file
  settings  Open settings
  features  Shows the status of experimental features

For more details on a specific command, pass it the help argument. [-?]

The following options are available:
  -v,--version  Display the version of the tool
  --info        Display general info of the tool

More help can be found at: https://aka.ms/winget-command-help
```

[1] https://github.com/microsoft/winget-cli/releases/tag/v-0.2.10191-preview

**Comment by quangkieu** [comment, marked as answer]
@neeraj9 https://github.com/microsoft/winget-cli/discussions/727#discussioncomment-328809 This should help

**Reply by neeraj9** [reply]
Thanks @quangkieu it helps.
